app/views/docs/email-and-sms-templates.phtml

```diff
@@ -0,0 +1,145 @@
+    <li>Under the <b>Auth</b> service, navigate to the <b>Templates</b> tab.</li>
+    <li>Expand the email template you want to edit.</li>
+    <li>Select the <b>Template language</b>. You can have a different template for each language your app supports.</li>
+    <li>Update the email template fields and click **Update** to save your changes.</li>
+</ol>
+<h2><a href="#email-templates" id="email-templates">Email Templates</a></h2>
+<p>You can customize the email emplates for account verification, magic-url authentication, password resets, and user invites.</p>
+<h3><a href="#email-template-components" id="email-template-components">Email Template Components</a></h3>
+<p>Each email templates has the following components that you can customize.</p>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 200px">Component</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>Sender name</td>
+        <td>Readers will see this as display name of the sender.</td>
+    </tr>
+    <tr>
+        <td>Sender email</td>
+        <td>Readers will see this as the displayed email of the sender. This must be a valid email for the SMTP provider your configured.</td>
+    </tr>
+    <tr>
+        <td>Reply to</td>
+        <td>Readers will reply to this email adress instead of the sender address. You can leave this field empty if unused.</td>
+    </tr>
+    <tr>
+        <td>Subject</td>
+        <td>The title of the email.</td>
+    </tr>
+    <tr>
+        <td>Message</td>
+        <td>The body of the email. You can find the variables available in the <a href="#email-template-syntax">Email Template Syntax</a> section.</td>
+    </tr>
+    </tbody>
+</table>
+<h3><a href="#email-template-syntax" id="email-template-syntax">Email Template Syntax</a></h3>
+<p>Variables can be used in email templates to dynamically format unique emails for each reader. These variables can only be accessed in the <b>Message</b> field of the email template.</p>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 200px">Variable</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>{{project}}</code></td>
+        <td>The project name.</td>
+    </tr>
+    <tr>
+        <td><code>{{team}}</code></td>
+        <td>Thee project team's name.</td>
+    </tr>
+    <tr>
+        <td><code>{{user}}</code></td>
+        <td>The name of the user receiving the email.</td>
+    </tr>
+    <tr>
+        <td><code>{{redirect}}</code></td>
+        <td>The URL for the user to complete the email template's action.</td>
+    </tr>
+    </tbody>
+</table>
+
+<h3><a href="#email-template-examples" id="email-template-components">Email Template Components</a></h3>
+[TODO: @dlohani once we're ready, let's do examples for each type of template (maybe also what it would look like rendered)]
```

Comment:
What do you mean by Email Template components?

Reply:
If its' console UI, then Arman already has it.

Reply:
> What do you mean by Email Template components?

Sorry, I mean email template **Examples**

Reply:
it should be ready, you can check with @ArmanNik